script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -116,9 +116,9 @@ If you have something to say, then say it. After all we've been through, there's
 She's right.
 After I nod, I signal for Arcueid to come near me.
 <Page38>
-"What that? A private talk?"
+"What's that? A private talk?"
 <Page39>
-Arcueid strides towards me excitedly and full of anticipation.
+Arcueid strides towards me excitedly, full of anticipation.
```

Comment:
I like the original more here. The forced pause is like... i dunno how to phrase it, but it's the cadence I'd expect in an explanation rather than just a description if that makes sense.

Reply:
> Arcueid comes over to me, brimming with excitement and anticipation.

? Stride is a bit forceful and not sure the JP backs it up.

Reply:
works for me